packages/cli/README.md

```diff
@@ -14,11 +14,13 @@ synthql generate \
     # The path where types will be generated to
     --out=src \
     # The default schema to use
+    # Option alias: `--schema`
```

Comment:
Notice that most of the diff in this MR is just about this small cosmetic change. It makes it harder to review, and will make merging harder for the person who has to merge these changes back into their branch.